internal/api/utils.go

```diff
@@ -245,13 +246,46 @@ func UpdateMemoryFromBytesToMiB(jsonData datatypes.JSON) map[string]interface{}
 		return nil
 	}
 
+	hasMoreThanThreeDecimals := func(value float64) bool {
+		precision := 0.001
+		str := strconv.FormatFloat(value, 'f', -1, 64)
+		decimalPart := strings.Split(str, ".")
+		if len(decimalPart) > 1 {
+			return len(decimalPart[1]) > int(math.Log10(1/precision)+1)
+		}
+		return false
+	}
+	
+	truncateToThreeDecimalPlaces := func(value float64) float64 {
+		if hasMoreThanThreeDecimals(value) {
+			truncated := math.Trunc(value * 1000) // Pushes decimal by 3 places
+			return truncated / 1000
+		}
+		return value
+	}
+	
+	convertCPU := func(cpu map[string]interface{}) error {
```

Comment:
We are never returning the error so I think we can remove the return type.

Reply:
The linter complains, if there is no `return` statement.

Reply:
Not sure how return value of `hasMoreThanThreeDecimals` relates with `convertCPU`. 
Also, I tried to remove the return type - http://pastebin.test.redhat.com/1103896 and did not got any linting issue.

Reply:
![Screenshot from 2023-06-30 16-11-32](https://github.com/RedHatInsights/ros-ocp-backend/assets/6542058/0a78ee5d-1d47-4729-853c-0751bbe1b20d)

Reply:
I have the updated Go packages as well

Reply:
![Screenshot from 2023-06-30 16-38-59](https://github.com/RedHatInsights/ros-ocp-backend/assets/6542058/ab5d5ebf-97b7-4f5c-8d5b-a101e6c37c0e)

Reply:
can you run `make lint` and check that way. Maybe IDE plugin issue.

Reply:
I tried make lint, no luck.
AFAIK function literals need a return type and return statement.